packages/lesswrong/server/callbacks/userCallbacks.tsx

```diff
@@ -501,3 +502,11 @@ getCollectionHooks("Users").editSync.add(function syncProfileUpdatedAt(modifier,
   }
   return modifier;
 });
+
+getCollectionHooks("Users").editAsync.add(async function updatingPostAudio(newUser: DbUser, oldUser: DbUser) {
+  const deletedChanged = newUser.deleted !== oldUser.deleted;
+  const nameChanged = newUser.displayName !== oldUser.displayName;
+  if (nameChanged || deletedChanged) {
```

Comment:
I would check for the presence of API keys here. No need to warn later on if they aren't configured.